verification/src/http_server/handlers/verification/solidity/multi_part.rs

```diff
@@ -1,4 +1,11 @@
-use super::types::{MultiPartFiles, VerificationRequest};
+use std::str::FromStr;
+
+use actix_web::{error, Error};
+use paperclip::actix::{
+    api_v2_operation,
+    web::{self, Json},
```

Comment:
`paperclip` is using proxy modules, so we have to include everywhere `paperclip::actix::web` instead of  `actix_web::web`